### ISSUE TITLE
Unable to setup breadcrumb for certain route paths

### ISSUE THREAD

**Issue by RFMV34** [udayvunnam/xng-breadcrumb#148]
Hi, I have this route structure
```
{
    path: 'feedbackChannels',
    children: [
      {
        path: '',
        component: ChannelsComponent,
        resolve: {
          channels: FeedbackChannelsResolve,
        },
        data: {
          title: 'Feedback Channel',
          breadcrumb: 'Feedback channels'
        }
      },
      {
        path: 'new',
        component: ChannelEditComponent,
        data: {
          breadcrumb: 'New',
          title: 'Feedback Channel'
        },
        resolve: {
          lookupEntries: LookupEntryCollectionResolve,
          sources: SourcesResolve,
        }
      },
    ],
  }
```
Breadcrumb for ChannelsComponent component (route `feedbackChannels/new`) does not work, but for ChannelEditComponent works (route `feedbackChannels/new`). What is wrong here? I see only `feedbackChannels`, not my custom string `Feedback channels`. 

Thanks, for help.

**Comment by syxov** [comment]
Hi @udayvunnam. As PR for this bug is merged, do you have plans to make a new version of the lib?

**Comment by udayvunnam** [comment]
@syxov thanks for the PR. The fix is released with v10.0.1

**Comment by udayvunnam** [comment]
The fix in the PR is causing a lot of other issues. I am reverting this until we find a proper folution
